.storybook/main.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  "stories": [
+    "../src/**/*.stories.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+ 
```

Comment:
```suggestion
 "addons": [
    "@storybook/addon-essentials",
  ],
```

You still need addon-essentials 🙂. This broke Storybook, so please make sure to test the impact of changes you make.

Reply:
Ok, I fix it right now.